content/en/users/security/secrets-store/_index.md

```diff
@@ -1,19 +1,49 @@
 ---
-title: Secrets Store
+title: EGI Secrets Store
 linkTitle: Secrets Store
 type: docs
-draft: true
 weight: 60
 description: >
-  Storing secrets in the EGI Cloud
+  Managing secrets in the EGI infrastructure
 ---
 
 ## What is it?
 
-The EGI Secrets Store helps you to **securely encrypt, store, and retrieve
-credentials for your services**. Instead of hardcoding credentials in your
-applications, you can make calls to Secrets Store to retrieve your credentials
-whenever needed.
+Your applications and services may need different secrets (credentials, tokens,
+passwords, etc.) during deployment and operation. These secrets are often
+stored as clear texts in code repositories or configuration files, which
+poses security risks. Furthermore, the secrets stored in files are static and
+difficult to change (rotate).
 
-The Secrets Store helps you protect access to your IT resources and data by
-enabling you to rotate and manage access to your secrets.
+The EGI Secrets Store helps you to
+**securely store, retrieve, and rotate credentials for your services**.
+
+The main features of EGI Secrets Store:
+
+- Simplifies, standardizes and secures the way users store secrets,
```

Comment:
```suggestion
- Simplify, standardize and secure the way users store secrets,
```

Reply:
Does not fit with the rest of the bullet points. Either all verbs/actions or all nouns/features.